src/main/java/com/stripe/model/CashBalance.java

```diff
@@ -49,54 +48,19 @@ public class CashBalance extends ApiResource {
   BalanceSettings settings;
 
   /** Retrieves a customer’s cash balance. */
-  public static CashBalance retrieveCashBalance(String customer) throws StripeException {
```

Comment:
This seems to not be a straight-forward removal

Reply:
I see the description. Please update the PR's title :)

Reply:
Good catch! Updated.